socs/agent/smurf_recorder.py

```diff
@@ -275,13 +297,20 @@ def read_frames(self, timeout=5):
 
             # Discard all flow control frames
             self.frames = [x for x in self.frames if 'sostream_flowcontrol' not in x]
-
+            # Discard Pipeline info frame
+            self.frames = [x for x in self.frames
+                           if x.type != core.G3FrameType.PipelineInfo]
+            if self.frames and not self.data_received:
+                self.data_received = True
+                self.log.info("Frames received from {addr}", addr=self.address)
```

Comment:
Since we'll only see this at the start of receiving frames, not every time frames are received, can we change the message to reflect that? That's to avoid someone looking at the logs, seeing only one line of "Frames received from {addr}" and no more, thinking that frames aren't still coming in. Maybe something like "Started receiving frames from {addr}".